docs/api/toast.md

```diff
@@ -22,78 +22,29 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
 A Toast is a subtle notification commonly used in modern applications. It can be used to provide feedback about an operation or to display a system message. The toast appears on top of the app's content, and can be dismissed by the app to resume user interaction with the app.
 
-## Presenting
+## Inline Toasts (Recommended)
```

Comment:
The Stackblitz examples for Angular and JS aren't working for any of the inline toast examples; Stackblitz issue possibly?

Reply:
Yeah currently StackBlitz is always pulling the latest Ionic, so unfortunately those won't work until the issue is resolved. @sean-perkins I believe is in touch with their team trying to get that fixed